codegen/module_system.go

```diff
@@ -618,6 +638,19 @@ func (g *tchannelClientGenerator) Generate(
 
 	sort.Sort(clientSpec.ModuleSpec.Services)
 
+	var clientQPSLevels map[string]string = make(map[string]string)
+	for _, methodName := range exposedMethods {
```

Comment:
Since we are doing this in multiple places, can we abstract this into a function?